packages/dev-frontend/src/components/Bonds/views/managing/WithdrawPane.tsx

```diff
@@ -75,22 +75,33 @@ export const WithdrawPane: React.FC = () => {
     setOutput(checkOutput(e.target.value));
 
   const handleConfirmPressed = () => {
+    const curveSlippage = 0.001; // Allow mininum of %0.1% slippage due to Curve rounding issues
     if (output === "both") {
+      const minBLusdAmount = withdrawal.get(BLusdAmmTokenIndex.BLUSD)?.mul(1 - curveSlippage);
+      const minLusdAmount = withdrawal.get(BLusdAmmTokenIndex.LUSD)?.mul(1 - curveSlippage);
+
+      if (minBLusdAmount === undefined || minBLusdAmount === Decimal.ZERO) return;
```

Comment:
Under which conditions could this happen? Would the user get any error?

Reply:
In theory it shouldn't happen since `withdrawal` should be initialized by the time `handleConfirmPressed` is invoked, however this check ensures that for whatever potential reason `withdrawal` is zero (e.g. a future dev mistake, or unknown bug), this function won't let the user withdraw with a 0 value.